src/Components/Button/consts.ts

```diff
@@ -4,10 +4,16 @@ export enum ButtonType {
   SECONDARY_GRAY = "secondaryGray",
   TERTIARY = "tertiary",
   TERTIARY_GRAY = "tertiaryGray",
+  PRIMARY_XLARGE = "primarty xlarge",
+  SECONDARY_GRAY_XLARGE = "secondaryGray xlarge",
+  CIRCULAR = "circular",
+  ButtonType = "ButtonType",
```

Comment:
1. all enums values should be in camelcase! 
2. typo: `  PRIMARY_XLARGE = "primarty xlarge"`